grails5 upgrade

niravassar opened the pull request.

Commit:
grails5 upgrade

sdelamo commented on complete/build.gradle:
```diff
@@ -6,10 +6,8 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
```
this should be decoupled from grails version

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,3 +1,6 @@
-grailsVersion=3.3.2
-gormVersion=6.1.8.RELEASE
-gradleWrapperVersion=3.5
+grailsVersion=5.0.0
```
grails version should be 5.0.1

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,3 +1,6 @@
-gormVersion=6.1.8.RELEASE
-gradleWrapperVersion=3.5
+grailsVersion=5.0.0
+gormVersion=7.1.0.M5
```
this should be GORM 7.1.0

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,3 +1,6 @@
-grailsVersion=3.3.2
-gormVersion=6.1.8.RELEASE
-gradleWrapperVersion=3.5
+grailsVersion=5.0.0
```
this should be decoupled from grails version

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,3 +1,6 @@
-gormVersion=6.1.8.RELEASE
-gradleWrapperVersion=3.5
+grailsVersion=5.0.0
+gormVersion=7.1.0.M5
```
this should be GORM 7.1.0

jamesfredley committed:
Modernize CI to Java 21 + Gradle 9 baseline (refs apache/grails-static-website#354)

jamesfredley committed:
Revert "Modernize CI to Java 21 + Gradle 9 baseline (refs apache/grails-static-website#354)"

This reverts commit fff730e53f5c770d971bb5da6ad8636c7d608975.

jamesfredley committed:
Update README to point at canonical guide on grails.apache.org (refs apache/grails-static-website#354)

jamesfredley committed:
Replace dead publisher CI with sample-app gradle check workflow

Delete: githubactions-build.sh
Add: .github/workflows/gradle.yml (auto-detects Java from Gradle wrapper version)

(refs apache/grails-static-website#354)
